torch/_logging/_internal.py

```diff
@@ -0,0 +1,359 @@
+import logging
```

Comment:
Can you turn on UFMT for this file (and the other new files you added). Add it to .lintrunner.toml